hedera-node/hedera-app-spi/src/main/java/module-info.java

```diff
@@ -2,6 +2,11 @@
     requires com.hedera.hashgraph.protobuf.java.api;
     requires static com.github.spotbugs.annotations;
 
+    // Swirlds requirements are due to the API in SchemaRegistry, specifically, SoftwareVersion.
+    // As the hashgraph platform modules are refactored, it may be that we can reduce the number
+    // of required imports
+    requires com.swirlds.common;
```

Comment:
Ah, we already have a services-specific versioning class called `SemanticVersion` which is part of the protobuf schema. I can use that instead of exposing `SoftwareVersion`.